specs/IsSmartScreenEnabled

```diff
@@ -0,0 +1,87 @@
+# Background
+
+[Edge SmartScreen](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-security-smartscreen) helps you identify reported phishing and malware websites,also helps you make informed decisions about downloads.
+
+In the current version of the Edge browser, users can switch the state of SmartScreen at any time. However, the webview2 application can only be set at initialization. In order to add more flexibility to the webview2 application. This API is added to keep the SmartScreen setting consistent with the browser. Development Users can control the status of SmartScreen at any time.
+
+In this document we describe the new setting. We'd appreciate your feedback.
+
+
+# Description
+The configuration of SmartScreen is stored in the user data folder. So its configuration is bound to the environment. Users can use ICoreWebView2Environment to control it.
+
```

Comment:
If this is stored in the user data folder, does that mean changes to the property persist across CoreWebView2Environment creations? Or does it always default to true?

Like if you create a CoreWebView2Environment, create a CoreWebView2 from it, the IsSmartScreenEnabled property will be true. If you change IsSmartScreenEnabled to false, close and dispose the environment and webview2, and then create a new CoreWebView2Environment and a new CoreWebView2, will the IsSmartScreenEnabled property be false because the value was stored in the user data folder or true because it always defaults to true?

Also, I assume that if two different host apps connect to the same browser process / user data folder, the IsSmartScreenEnabled property value will still be correct when set by one and read by the other?

Reply:
Oh OK I misunderstood but you've updated the text so now it doesn't look to me like we serialize the value of IsSmartScreenEnabled in the user data folder across WebView2 sessions.